# Nikolaus Wagner (C) 2023
# nwagner@lincoln.ac.uk

std_msgs/String model_name
---
std_msgs/Bool success